components/Modal/constants/src/index.ts

```diff
@@ -1,21 +1,21 @@
 export const ClassBackDrop =
-  'fixed left-0 top-0 z-[1000010] h-screen w-screen bg-gray-900/[.92] backdrop-blur'
+  'fixed left-0 top-0 z-[1000010] h-screen w-screen bg-gray-900/[.80] backdrop-blur-sm'
 export const ClassModalContainer =
   'pointer-events-none fixed left-0 top-0 z-[1000020] h-full w-screen overflow-hidden flex items-center justify-center'
 export const ClassModal =
-  'pointer-events-auto relative flex flex-col rounded-lg border-4 border-gray-700 bg-white'
+  'pointer-events-auto relative flex flex-col rounded outline outline-4 outline-gray-500/[.40] bg-white'
 export const ClassModalStandardDimensions =
-  'max-h-[80vh] min-w-[200px] sm:min-w-[390px] sm:max-w-[70vw] shrink-0'
+  'max-h-[80vh] min-w-[200px] sm:min-w-[390px] max-w-[calc(100vw-32px)] sm:max-w-[70vw] shrink-0'
 export const ClassModalFullscreenDimensions =
   'h-[calc(100vh-32px)] sm:w-[calc(100vw-32px)]'
 export const ClassTitleBox =
-  'flex shrink-0 items-center justify-between rounded-t-md border-b border-gray-100 p-[16px] px-[24px]'
+  'flex shrink-0 items-center justify-between rounded-t-md border-b border-gray-100 p-[16px] px-16px sm:px-[24px]'
 export const ClassTitle = 'text-[18px] font-medium leading-[28px] text-gray-900'
 export const ClassHelpLinkDash = 'mx-[8px] h-[1px] w-[32px] bg-gray-100'
 export const ClassHelpLink =
   'flex items-center text-[16px] font-normal leading-[24px] text-indigo-500'
 export const ClassCloseButton =
   'group rounded-full border-transparent p-[4px] outline-none'
```

Comment:
@elevatebart Could you help figure out why the hover state isn't actually working with the `transition` classes for the close icon here? It's probably going to be faster for you. The smooth transition isn't working though.